pkg/readychecker/types/built_in.go

```diff
@@ -6,19 +6,20 @@ import (
 
 	sc_v1b1 "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
 	"github.com/pkg/errors"
-	apps_v1b2 "k8s.io/api/apps/v1beta2"
+	apps_v1 "k8s.io/api/apps/v1"
 	core_v1 "k8s.io/api/core/v1"
 	ext_v1b1 "k8s.io/api/extensions/v1beta1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
 var MainKnownTypes = map[schema.GroupKind]readychecker.IsObjectReady{
-	{Group: core_v1.GroupName, Kind: "ConfigMap"}:    alwaysReady,
-	{Group: core_v1.GroupName, Kind: "Secret"}:       alwaysReady,
-	{Group: core_v1.GroupName, Kind: "Service"}:      alwaysReady,
-	{Group: apps_v1b2.GroupName, Kind: "Deployment"}: isDeploymentReady,
-	{Group: ext_v1b1.GroupName, Kind: "Ingress"}:     alwaysReady,
+	{Group: core_v1.GroupName, Kind: "ConfigMap"}:      alwaysReady,
```

Comment:
it sort of seems like this should just be something in the config and dynamically generated?
(i.e. a list of object types, regardless of whether they're core, Service Catalog, or whatever...)

Reply:
(then ask the cluster if they're supported...)

Reply:
Yes, I know :) One of the future things.